linux_os/guide/system/network/network-nftables/nftables_rules_permanent/ansible/sle15.yml

```diff
@@ -0,0 +1,19 @@
+# platform = SUSE Linux Enterprise 15
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+{{% set nftables_family_names = ['bridge', 'arp', 'inet'] %}}
```

Comment:
This could be removed if you use the Ansible loop in the `lineinfile` task.

Reply:
Thanks for the feedback :bow: reworked it in 5814f092f8

Reply:
You only missed to remove the list declaration here. : )
```suggestion
```